backend/src/main/resources/liquibase/create-index-changelog.xml

```diff
@@ -998,4 +998,34 @@
         </createIndex>
     </changeSet>
 
+    <changeSet id="11.1.2-index-projectname-workspace-workspacetype-deleted" author="anandJ">
+        <preConditions onFail="MARK_RAN">
+            <not>
+                <indexExists indexName="p_name_wspace_wspace_type_deleted"/>
+            </not>
+        </preConditions>
+        <createIndex indexName="p_name_wspace_wspace_type_deleted"
+                     tableName="project">
+            <column name="name" type="varchar(255)"/>
+            <column name="workspace" type="varchar(255)"/>
+            <column name="workspace_type" type="int"/>
+            <column name="deleted" type="boolean"/>
+        </createIndex>
+    </changeSet>
+
+    <changeSet id="11.2.2-index-datasetname-workspace-workspacetype_deleted" author="anandJ">
+        <preConditions onFail="MARK_RAN">
+            <not>
+                <indexExists indexName="d_name_wspace_wspace_type_deleted"/>
+            </not>
+        </preConditions>
+        <createIndex indexName="d_name_wspace_wspace_type_deleted"
+                     tableName="dataset">
+            <column name="name" type="varchar(255)"/>
+            <column name="workspace" type="varchar(255)"/>
+            <column name="workspace_type" type="int"/>
+            <column name="deleted" type="boolean"/>
+        </createIndex>
+    </changeSet>
+
```

Comment:
before this change, `deleted` column was created after this change sets so Liquebase gives the error with `column not found`. that's why moved here